src/scripts/scenes/xr-scene.js

```diff
@@ -67,11 +65,6 @@ export default class XrScene {
     this._animationCallback();
   }
 
-  _restartAnimation = () => {
```

Comment:
This was necessary to keep the animation loops from continuing when an XR session was started.